src/_assets/scss/inc/page.scss

```diff
@@ -81,7 +81,7 @@
 	.video-embed,
 	.embed-responsive {
 		width: 100%;
-		aspect-ratio: 16 / 9;
+		@include mixins.aspectRatio(16, 9);
 
 		iframe {
```

Comment:
Doesn't the iframe need to be positioned absolutely within the ratio box if using the padding hack?

Reply:
Right, fix incoming :)

Reply:
Pushed an update.